specification/sql/resource-manager/readme.md

```diff
@@ -1888,6 +1887,7 @@ input-file:
 - ./Microsoft.Sql/preview/2023-05-01-preview/Servers.json
 - ./Microsoft.Sql/preview/2023-05-01-preview/ReplicationLinks.json
 - ./Microsoft.Sql/preview/2023-08-01-preview/DistributedAvailabilityGroups.json
+- ./Microsoft.Sql/preview/2024-11-01-preview/DatabaseOperations.json
```

Comment:
You have a tag 2024-11-01-preview https://github.com/Azure/azure-rest-api-specs/blob/main/specification/sql/resource-manager/readme.md#tag-package-preview-2024-11-01-preview which already version uniformly, why not add your APIs there? 
Another question is why not change your default tag to 2024-11-01-preview? It's the latest one.